llama-index-integrations/llms/llama-index-llms-gaudi/README.md

```diff
@@ -0,0 +1,18 @@
+# LlamaIndex Llms Integration with Intel Gaudi
+
+## Installation
+
+```bash
+pip install --upgrade-strategy eager optimum[habana]
+```
+
+## Usage
+
+```python
+from llama_index.llms.gaudi import GaudiLLM
```

Comment:
Should we show how to use the LLM beyond importing it here?

Reply:
Updated this README to include code example beyond imports.